src/pages/en/guides/cms/wordpress.md

```diff
@@ -2,12 +2,222 @@
 
+## Integrating with Astro
+
+WordPress comes with a built-in [WordPress REST API](https://developer.wordpress.org/rest-api/) to connect your WordPress data to Astro. You can optionally install [WPGraphQL](https://wordpress.org/plugins/wp-graphql/) on your site to use GraphQL.
+
+### Prerequisites
+
+To get started, you will need to have the following:
+
+1. **An Astro project** - If you don't have an Astro project yet, our [Installation guide](/en/install/auto/) will get you up and running in no time.
+
+2. **A WordPress site** - Your site's REST API is `[YOUR_SITE]/wp-json/wp/v2/` and is available by default with any WordPress site. It is also possible to [set up WordPress on a local environment](https://wordpress.org/support/article/installing-wordpress-on-your-own-computer/).
+
+### Setting up Credentials
+
+Your WordPress REST API is available to external requests for data fetching of your posts without authentication by default. This does not allow users to modify your data or site settings and allows you to use your data in your Astro project without any credentials.
+
+You may choose to [require authentication](https://developer.wordpress.org/rest-api/frequently-asked-questions/#require-authentication-for-all-requests) if necessary.
+
+### Installing Depenencies
+
+There are no dependencies required to fetch your WordPress data from Astro via the REST API.
+
+### Fetching Data
+
+You will fetch your WordPress data through your site's unique REST API URL and the route for your content. (For a blog, this will commonly be `posts`.) Then, you can render your post's properties using Astro's `set:html=""` directive. For example, to display a list of post titles and their content:
+
+```astro title="src/pages/index.html
+---
+import Layout from '../layouts/Layout.astro';
+const response = await fetch("https://[YOUR-SITE]/wp-json/wp/v2/posts")
+const allPosts = await response.json()
+---
+<Layout>
+<h1>My Posts</h1>
+{
+  allPosts.map((post) => (
+    <>
+      <h2 set:html="post.title.rendered" />
+      <p set:html="post.content.rendered" />
+    </>
+  ))
+}
+</Layout>
+```
+
+The WordPress REST API includes [global parameters](https://developer.wordpress.org/rest-api/using-the-rest-api/global-parameters/) such as `_fields` and `embed`. 
+
+A large quantity of data is available to you via this API, so you may wish to only fetch certain fields. You can restrict your response by adding the `_fields` paramater to the API URL, for example: `[YOUR-SITE]/wp/v2/posts?_fields=author,id,excerpt,title,link` 
+
+The API can also return content related to your post, such as a link to the parent post, or to comments on the post. You can add the `_embed` parameter to the API URL (e.g. `[YOUR-SITE]/wp/v2/posts?_embed`) to indicate to the server that the response should include these embedded resources.
```

Comment:
`_embed` does not *need* any params. I think it can if you want to restrict to only fetching certain embedded resources, according to what I read about the api!